cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -389,6 +392,39 @@ private static File hijackYarnConfFile(File confFile) {
   }
 
   /**
+   * Check that the file is a mapred-site.xml file, and return a temp copy of it to which are added
+   * necessary options. If it is not a mapred-site.xml file, return it as is.
```

Comment:
this comment is not valid, as we are not doing those checks here. i know its from other method, can you change it?